### PR TITLE
Make fabricators deconstructable without head of staff access

### LINKED ISSUE #7250: Robotics fab cant be deconned with deconstruction device
### Labels

[MINOR]

### Description

Even though im a roboticist, cut the access wires and all that, scanned my card, everything, it says i dont have permission to deconstruct it. I am 99% sure this is a bug.

### Steps to reproduce

1.get a deconstruction device
2.attempt to deconstruct robotics fab
3.be sad that it wont work

### Expected Behavior

to be able to deconstruct it so i can have my own robotics area in peace

### Additional Information & Screenshots

Reported by: AireDrew
On server: Goonstation 1 Classic: Heisenbee
Round log date: 2022-01-22-03-55
Reported on: 2022-01-22 04:00:12

### PR DESCRIPTION
**Pull request by stonepillars**
[BUG]

## About the PR 

Makes fabricators deconstructable, specifically by giving them the `DECON_BUILT` decon flag

## Why's this needed? 

Fixes #7250 

image

### CODE REVIEW

File: code/obj/machinery/manufacturer.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	mats = 20
 	req_access = list(access_heads)
 	event_handler_flags = NO_MOUSEDROP_QOL
-	deconstruct_flags = DECON_SCREWDRIVER | DECON_WRENCH | DECON_WELDER | DECON_WIRECUTTERS | DECON_MULTITOOL
+	deconstruct_flags = DECON_SCREWDRIVER | DECON_WRENCH | DECON_WELDER | DECON_WIRECUTTERS | DECON_MULTITOOL | DECON_BUILT
```
Comment (Azrun) [inline code/obj/machinery/manufacturer.dm:14]: Add a comment about why flag is added.

**Review by Azrun — CHANGES_REQUESTED** [review]
Don't think changelog is needed as this is a bug.

**Review by Azrun — APPROVED** [review]

**Comment by zjdtmkhzt** [comment]
Does this make sense though? The fabricators aren't player built by default, so why would they get this flag. Shouldn't we fix the access check so it works properly for fabricators instead?

**Comment by Azrun** [comment]
With hindsight I agree zjdtmkhzt.  At the time we incorrectly assumed this would result in the desired behavior (which it technically did) but had unforeseen consequences... like rkit/gene-booth locking PR. 😢

**Pull request merged**